lib/utils.py

```diff
@@ -177,11 +175,16 @@ def recursive_glob(directory, pattern):
     """
 
     matches = []
-    for root, _, filenames in os.walk(directory):
+
+    for root, _, filenames in os.walk(directory, onerror=base_directory_not_found_error):
         for filename in fnmatch.filter(filenames, pattern):
             matches.append(os.path.join(root, filename))
+
     return matches
 
+def base_directory_not_found_error(err):
+    if err.errno == errno.ENOENT:
+        raise exception.FilesToValidateNotFound()
```

Comment:
To re-raise the exception, shoud I just put `raise err` at the end of the function then?